java-sdk/pom.xml

```diff
@@ -6,7 +6,7 @@
         <groupId>org.sagebionetworks.bridge</groupId>
         <artifactId>sdk-group</artifactId>
         <!-- Please use the maven versions plugin to manage this, e.g.`mvn versions:set -DnewVersion=0.8.1`-->
-        <version>0.10.3</version>
+        <version>0.10.11</version>
```

Comment:
oh its my mistake.. i update the version every time i need to change after mvn install the sdk.. Do you know any other method to force upload jar file in integ test?